docs/configuration/flagd_start.md

```diff
@@ -15,7 +15,9 @@ flagd start [flags]
   -e, --evaluator string                    DEPRECATED: Set an evaluator e.g. json, yaml/yml.Please note that yaml/yml and json evaluations work the same (yaml/yml files are converted to json internally) (default "json")
   -h, --help                                help for start
   -z, --log-format string                   Set the logging format, e.g. console or json  (default "console")
+  -t, --metrics-exporter string             Set the metrics exporter. Default(if unset) is Prometheus. Can be override to otel - OpenTelemetry metric exporter. Overriding to otel require otelCollectorTarget to be present
   -m, --metrics-port int32                  Port to serve metrics on (default 8014)
+  -o, --otel-collector-target string        Set the grpc target of the otel collector for flagd runtime. If unset, collector setup will be ignored resulting in ignored traces(NoopTracerProvider)
```

Comment:
```suggestion
  -o, --otel-collector-uri string        Set the grpc URI of the OpenTelemetry collector for flagd runtime. If unset, the collector setup will be ignored and traces will not be exported.
```

Reply:
went for `target` as this is kind of the grpc terminology at API level [1] , but yes let's go for `uri`

[1] - https://pkg.go.dev/google.golang.org/grpc#Dial